GW2EIEvtcParser/EncounterLogic/Strikes/EndOfDragons/HarvestTemple.cs

```diff
@@ -1157,5 +1799,45 @@ private static bool CustomCheckVoidwalkerEligibility(ParsedEvtcLog log)
             }
             return true;
         }
+
+        /// <summary>
+        /// Matches an effect to another effect by proximity and filters out additional effects.
+        /// </summary>
+        /// <param name="startEffects">List of the initial effects for the positions.</param>
+        /// <param name="endEffects">List of the final effects for the positions.</param>
+        /// <returns>Filtered list with matched <paramref name="startEffects"/>, <paramref name="endEffects"/> and distance between them.</returns>
+        private static List<(EffectEvent, EffectEvent, float)> MatchEffectToEffect(IReadOnlyList<EffectEvent> startEffects, IReadOnlyList<EffectEvent> endEffects)
+        {
+            var matchedEffects = new List<(EffectEvent, EffectEvent, float)>();
```

Comment:
added names to the tuple: startEffect, endEffect, distance
Would be much clearer to read than Item1, Item2 and Item3